packages/core-database/lib/wallet-manager.js

```diff
@@ -184,18 +185,61 @@ module.exports = class WalletManager {
     Object.values(this.byAddress).map(wallet => (wallet.dirty = false))
   }
 
+  /**
+   * Load a list of all active delegates.
+   * @param  {Number} maxDelegates
+   * @return {Array}
+   */
+  activeDelegation (maxDelegates, height) {
+    if (height > 1 && height % maxDelegates !== 1) {
+      throw new Error('Trying to build delegates outside of round change')
+    }
+
+    const { round } = roundCalculator.calculateRound(height, maxDelegates)
+    let delegates = this.allByUsername()
+
+    if (delegates.length < maxDelegates) {
+      throw new Error(`Expected to find ${maxDelegates} delegates but only found ${delegates.length}. This indicates an issue with the genesis block & delegates.`)
+    }
+
+    delegates = delegates.sort((a, b) => {
+      const aBalance = +a.voteBalance.toFixed()
+      const bBalance = +b.voteBalance.toFixed()
+
+      if (aBalance === bBalance) {
+        // logger.warn(`Delegate ${a.username} (${a.publicKey}) and ${b.username} (${b.publicKey}) have a matching vote balance of ${a.voteBalance.dividedBy(Math.pow(10, 8)).toLocaleString()}.`)
+
+        if (a.publicKey === b.publicKey) {
+          logger.error(`The balance and public key of both delegates are identical! Delegate "${a.username}" appears twice in the list.`)
+        }
+
+        return a.publicKey.localeCompare(b.publicKey, 'en-US-u-kf-lower')
```

Comment:
why this 'en-US-u-kf-lower' ? 'en' should be ok as there is only '0123456789abcdef' as letters

Reply:
Was mainly to make sure the string is treated as a lowercase string in case there ever is a mixed-case public key for whatever reason but changed it to `en`.